lib/remap-lang/grammar.pest

```diff
@@ -80,6 +81,7 @@ regex_flags =  { ("i" | "x" | "m")* }
 
 // Other ----------------------------------------------------------------------
 
+conditional_expr = { "(" ~ NEWLINE* ~ (expression ~ (NEWLINE+ | ";"))* ~ boolean_expr ~ NEWLINE* ~ ")" }
```

Comment:
I'm not sure this is quite right. The final `NEWLINE*` in this doesn't seem to be picked up. So an expression like the following:

```
if (
   $$ook="nonk"; .plork == 2
   ) {
```

results in the following error:

```
Dec 07 16:46:50.288 ERROR vector::topology: Configuration error. error=Transform "remap": remap error: parser error:  --> 3:13
  |
3 |             ) {␊
  |             ^---
  |
  = expected assignment, if_statement, not, or block
```

Do you have any ideas why this might be?